### PR TITLE
Updated Impeller details

### PR DESCRIPTION
**Pull request by atsansone**
Fixes https://github.com/flutter/website/issues/8608

### CODE REVIEW

File: src/perf/impeller.md [hunk 1]
```diff
@@ -1,152 +1,111 @@
...
 ### iOS
 
-Impeller is currently available for preview on iOS.
-You can enable it by passing `--enable-impeller`
-to the `flutter run` command or add the
-following under the top-level `<dict>` tag in an
-application's `Info.plist` file:
+Flutter enables Impeller by default on iOS.
+
+To _disable_ Impeller on iOS, take one of the following actions:
```
Comment (jonahwilliams) [inline src/perf/impeller.md:53]: Worth noting that --disable-impeller is only for debugging and if you want to _ship_ with impeller off you must add it to the Plist entry.

**Review by jonahwilliams — APPROVED** [review]
LGTM

File: src/perf/impeller.md [hunk 2]
```diff
@@ -1,152 +1,111 @@
...
 
-## Overview
+## What is Impeller?
 
-What is Impeller?
-
-Impeller is a new rendering runtime for Flutter and
-is the Flutter team's solution to Flutter's
+Impeller provides a new rendering runtime for Flutter.
+The Flutter team's believes this solves Flutter's
```
Comment (loic-sharma) [inline src/perf/impeller.md:9]:
```suggestion
The Flutter team believes this solves Flutter's
```

File: src/perf/impeller.md [hunk 3]
```diff
@@ -1,152 +1,111 @@
...
-These minor differences might (or might not) be bugs,
-so don’t hesitate to [file an issue][file-issue]
-with a small reproducible test case, and include
-`[Impeller]` in the title.
-
-On Android, the team continues to make progress on a Vulkan
-backend for Impeller. Impeller isn't yet ready for preview on Android,
-but stay tuned for more progress in a future release.
+Android development continues but it's not ready for preview.
+On the `master` channel, it might not work at the tip-of-tree.
```
Comment (loic-sharma) [inline src/perf/impeller.md:80]: It looks like this sentence got garbled. Maybe consider:

```suggestion
Impeller on Android might not work on the `master` channel.
```

**Review by loic-sharma — APPROVED** [review]
LGTM but not very familiar with Impeller :)

File: src/perf/impeller.md [hunk 4]
```diff
@@ -1,152 +1,111 @@
...
-For detailed information about Impeller's design
-and architecture, check out the [README.md][]
-file in the source tree.
+To learn more details about Impeller's design and architecture,
+check out the [README.md][] file in the source tree.
 
 [README.md]: {{site.github}}/flutter/engine/blob/main/impeller/README.md
 
-## Documentation, references, and additional reading
+## Additional Information
```
Comment (sfshaza2) [inline src/perf/impeller.md:108]: Should use sentence case.

**Pull request merged**